### DISCUSSION TITLE
A question from old memory :)

### DISCUSSION THREAD

**Discussion by rastaman111** [LeoNatan/LNPopupController#574] [General] [answered]
I remember that old versions of the pod required to implement this SettingsTableViewController file into the project, is it now out of date or is it still required?

**Comment by LeoNatan** [comment]
Do not open issues when it’s clearly a question.
And I have no idea what you are asking. What “pod”? CocoaPods? LNPopupController has never supported CocoaPods.

**Comment by rastaman111** [comment]
Well, I didn't talk about CocoaPods. I talked about the SettingsTableViewController .h and .m files.

**Reply by iDevelopper** [reply]
@rastaman111,

You mentioned Pod though!

**Reply by LeoNatan** [reply]
Those are demo project files. What do they have to do with LNPopupController. I still don’t understand what you are asking.

**Reply by rastaman111** [reply]
> Those are demo project files. What do they have to do with LNPopupController. I still don’t understand what you are asking.

When I first learned about your library, and it was 2019. I installed it and it required this file. In subsequent projects, I immediately transferred this file.

**Reply by rastaman111** [reply]
> @rastaman111,
> 
> You mentioned Pod though!

When I say Pod I mean a library

**Reply by LeoNatan** [reply, marked as answer]
No, you don’t need any of the demo project files. It is a separate project in the repo, just for testing and demonstrating the framework. If you are using Swift Package Manager, they are excluded automatically for you. If you are using submodules or whatever other way to bring the framework, add the LNPopupController.xcodeproj as a subproject to your project in Xcode, and link against LNPopupController.framework.
